packages/connect/src/Detector.test.ts

```diff
@@ -1,103 +1,59 @@
-      detect = new Detector('test-uapp');
-      const api = await detect.connect('westend');
-      expect(api).toBeTruthy();
-      done();
-    } catch (err: unknown) {
-      done(err);
-    } finally {
-      await detect.disconnect('westend');
-    }
+  test('Should connect with known chain "westend".', async () => {
+    detect = new Detector('test-uapp');
+    const api = await detect.connect('westend');
+    expect(api).toBeTruthy();
+    await detect.disconnect('westend');
   }, 15000);
 
-  test('Should connect with known chain "polkadot".', async done => {
-    try {
-      detect = new Detector('test-uapp');
-      const api = await detect.connect('polkadot');
-      expect(api).toBeTruthy();
-      done();
-    } catch (err: unknown) {
-      done(err);
-    } finally {
-      await detect.disconnect('polkadot');
-    }
+  test('Should connect with known chain "polkadot".', async () => {
+    detect = new Detector('test-uapp');
+    const api = await detect.connect('polkadot');
+    expect(api).toBeTruthy();
+    await detect.disconnect('polkadot');
   }, 15000);
 
-  // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
-  test.skip('Should connect with known chain "kusama".', async done => {
-    try {
-      detect = new Detector('test-uapp');
-      const api = await detect.connect('kusama');
-      expect(api).toBeTruthy();
-      done();
-    } catch (err: unknown) {
-      done(err);
-    } finally {
-      await detect.disconnect('kusama');
-    }
+  test('Should connect with known chain westend, no chainSpecs and options', async () => {
+    const chainName = 'westend';
+    detect = new Detector('test-uapp');
+    const options = {} as ApiOptions;
+    const api = await detect.connect(chainName, undefined, options);
+    expect(api).toBeTruthy();
+    await detect.disconnect('westend');
   }, 15000);
 
-  test('Should connect with unknown chain westend2 and chainSpecs.', async done => {
-    try {
-      const chainSpec = JSON.stringify(westend2);
-      const chainName = 'westend2';
-      detect = new Detector('test-uapp');
-      const api = await detect.connect(chainName, chainSpec);
-      expect(api).toBeTruthy();
-      done();
-    } catch (err: unknown) {
-      done(err);
-    } finally {
-      await detect.disconnect('westend2');
-    }
-  }, 15000);
 
-  test('Should connect with unknown chain westend2, chainSpecs and options', async done => {
-    try {
-      const chainSpec = JSON.stringify(westend2);
-      const chainName = 'westend2';
-      detect = new Detector('test-uapp');
-      const options = {} as ApiOptions;
-      const api = await detect.connect(chainName, chainSpec, options);
-      expect(api).toBeTruthy();
-      done();
-    } catch (err: unknown) {
-      done(err);
-    } finally {
-      await detect.disconnect('westend2');
-    }
+  // DEACTIVATED for now due to chainSpecs - once Kusama specs are available this should be activated
```

Comment:
This says DEACTIVATED but the test is not ignored.

Reply:
afair kusama was fixed on the mean time - and test started to succeeded. Just did not remove the comment (which is now removed and test still pass)